packages/core/src/Provider/HandleProvider/types.ts

```diff
@@ -0,0 +1,42 @@
+import { Cardano, Point, Provider } from '../..';
+
+export type Handle = string;
+
+/**
+ * Type of addresses that can resolve from a Handle
+ */
+export type CardanoAddress =
+  | Cardano.ByronAddress
+  | Cardano.PaymentAddress
+  | Cardano.EnterpriseAddress
+  | Cardano.PointerAddress;
```

Comment:
This should no longer be used, please delete
```suggestion
```

Reply:
aa5eaf7df693d418c3cdb800b0e524e2e316ae28 should fix this

Reply:
@mkazlauskas can you point me to any prior discussion/justification on this decision? The `CardanoAddress` type is a domain expression

Reply:
Created a placeholder ticket LW-6784 so we can get back to this later